check/check.ml

```diff
@@ -324,6 +334,64 @@ module Exists = Make(struct
   let eqaf_false () = Eqaf.exists_uint8 ~f hash_neq_0
 end)
 
+module Find = Make(struct
```

Comment:
I tried to add linear regression tests for `find_uint8` and the new functions, but I don't think my additions work.
Help would be appreciated, I'm a bit lost in this section :-)

Reply:
Fixed 👍